syntax = "proto3";

message LocationMessage {
  string creation_time = 1;
  string longitude = 2;
  int32 person_id = 3;
  int32 id = 4;
  string latitude = 5;
}

message LocationID {
  int32 id = 1;
}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
    rpc Get(LocationID) returns (LocationMessage);
}